doozer/tests/test_image.py

```
@@ -728,8 +728,13 @@ def test_apply_alternative_upstream_config_no_alternative_config(
     @patch('doozerlib.image.SourceResolver')
     @patch('builtins.open', create=True)
     @patch('pathlib.Path.joinpath')
-    def test_determine_upstream_rhel_version_ubi_pattern(self, mock_joinpath, mock_open, mock_source_resolver):
+    @patch('doozerlib.image.util.oc_image_info_for_arch__caching', return_value={'config': {'config': {'Labels': {}}}})
+    def test_determine_upstream_rhel_version_ubi_pattern(
+        self, mock_oc_image_info, mock_joinpath, mock_open, mock_source_resolver
```

Ruff (0.15.0)

[warning] Unused method argument: `mock_oc_image_info`

(ARG002)